hermes-management/src/main/java/pl/allegro/tech/hermes/management/domain/topic/schema/SchemaSourceService.java

```diff
@@ -4,58 +4,63 @@
 import org.springframework.stereotype.Component;
 import pl.allegro.tech.hermes.api.SchemaSource;
 import pl.allegro.tech.hermes.api.Topic;
-import pl.allegro.tech.hermes.domain.topic.schema.SchemaVersion;
+import pl.allegro.tech.hermes.api.TopicName;
+import pl.allegro.tech.hermes.management.config.TopicProperties;
 import pl.allegro.tech.hermes.management.domain.topic.TopicService;
 import pl.allegro.tech.hermes.management.infrastructure.schema.validator.SchemaValidator;
 import pl.allegro.tech.hermes.management.infrastructure.schema.validator.SchemaValidatorProvider;
+import pl.allegro.tech.hermes.schema.SchemaSourceClient;
+import pl.allegro.tech.hermes.schema.SchemaVersion;
 
 import java.util.Optional;
 
-import static pl.allegro.tech.hermes.api.ContentType.AVRO;
 import static pl.allegro.tech.hermes.api.TopicName.fromQualifiedName;
 
 @Component
 public class SchemaSourceService {
```

Comment:
i wonder - we use term `schema source` everywhere and i don't know if it's a proper one - schema source sounds like source of schema aka schema repository, while what we mean is schema itself (aka raw schema in text format). Maybe it's a good time to change this?